superset-frontend/src/theme/index.ts

```diff
@@ -69,6 +69,12 @@ const baseConfig: ThemeConfig = {
       colorText: supersetTheme.colors.text.label,
       remainingColor: supersetTheme.colors.grayscale.light4,
     },
+    Slider: {
+      trackBgDisabled: supersetTheme.colors.grayscale.light1,
+      colorBgElevated: supersetTheme.colors.grayscale.light5,
+      handleSizeHover: supersetTheme.gridUnit * 2.5,
+      handleLineWidthHover: supersetTheme.gridUnit / 2,
```

Comment:
I wonder if we should tie the handle size and line width do grid units. It sounds like something that's independent of grid size

Reply:
Agreed